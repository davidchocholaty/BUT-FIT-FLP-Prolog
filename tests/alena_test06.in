S a B R
B a C R
C a D L
C a C R
D a E L
E a F q
aaa