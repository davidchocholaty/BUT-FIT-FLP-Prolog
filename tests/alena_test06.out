Saaa
aBaa
aaCa
aDaa
Eaaa
Fqaa
